web/templates/attendance/fill_course_attendance.html.eex

```diff
@@ -1,34 +1,99 @@
-<h2>Filling attendances for <%=@offered_course.course.name%></h2>
-<h3>Taught by : <%= get_teacher_display_name(@offered_course.teachers) %></h3>
 
-<%= form_for @changeset, attendance_update_fill_path(@conn, :update_fill, @offered_course), fn f -> %>
+  <% title = Enum.join(["Fill attendances for", @offered_course.course.name, "in", @offered_course.term.name], " ") %>
 
-<table class="table">
-  <thead>
-    <tr>
-      <th>Class Date</th>
-      <th>Class Time</th>
-      <%= for student <- @offered_course.students do %>
-        <th><%= get_student_display_name(student) %></th>
-      <% end %>
-    </tr>
-  </thead>
-  <tbody>
-    <%= inputs_for f, :classes, fn ch -> %>
-      <tr>
-        <td><%= ch.data.date %></td>
-        <td><%= ch.data.starting_at %></td>
-        <%= inputs_for ch, :attendances, fn fh -> %>
-          <td><%= select(fh, :attendance_type, ["Present", "Absent", "Not filled"], selected: fh.data.attendance_type) %></td>
+  <div class="row">
+    <div class="
+      col-xs-12
+      col-sm-offset-1 col-sm-10
+      col-md-offset-2 col-md-8
+      col-lg-offset-2 col-lg-8
+    ">
+      <%= form_for @changeset, attendance_update_fill_path(@conn, :update_fill, @offered_course), fn f -> %>
+        <%= CoursePlanner.SharedView.card title do %>
+          <%= CoursePlanner.SharedView.card_content do %>
+            <%= if @changeset.action do %>
+              <div class="form-error">
+                <p>Oops, something went wrong! Please check the errors below.</p>
+              </div>
+            <% end %>
+            <div class="row">
+              <div class="col-xs-5 col-sm-3 col-md-3">
+                <table class="attendance-date-table">
+                  <thead>
+                    <tr class="attendance-date-table__thead-tr">
+                      <th class="attendance-date-table__th">Class Date</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    <%= for class <- @offered_course.classes do %>
+                      <tr class="attendance-date-table__row">
+                        <td class="attendance-date-table__col">
+                          <%= class.date %>
+                          <%= String.slice Ecto.Time.to_iso8601(class.starting_at), 0, 5 %>
```

Comment:
You can use a formatter like `Timex.format(class.starting_at, "{h24}:{m}")` for better formatting, also in case you want to change in the future.